presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -349,4 +353,31 @@ public IcebergConfig setSplitManagerThreads(int splitManagerThreads)
         this.splitManagerThreads = splitManagerThreads;
         return this;
     }
+
+    @Min(0)
+    public int getMetadataPreviousVersionsMax()
+    {
+        return metadataPreviousVersionsMax;
+    }
+
+    @Config("iceberg.metadata-previous-versions-max")
+    @ConfigDescription("The max number of previous metadata files exist in metadata log")
```

Comment:
Nit : Would it make more clear if we has a description `The max number of old metadata files to keep` since this is something now user can control with this config property.

Reply:
Sure, it seems better, I have changed here as well as the corresponding description in `IcebergTableProperties` and `docs`. Please take a look when convenient. @agrawalreetika @steveburnett @ZacBlanco